### PR TITLE
VIDEO-4958 Consuming the changes in twilio-webrtc.js PR #141.

### PR DESCRIPTION
**Pull request by manjeshbhargav**
@makarandp0 @PikaJoyce 

This PR consumes the changes made in twilio/twilio-webrtc.js#141.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -145,7 +145,7 @@
     "clean": "rimraf ./coverage ./es5 ./dist"
   },
   "dependencies": {
-    "@twilio/webrtc": "4.4.0",
+    "@twilio/webrtc": "github:twilio/twilio-webrtc.js#49f022079ec4d57bf41b71a414845c2ef0dcb121",
```
Comment (PikaJoyce) [inline package.json:148]: Should this be `5.0.0` by chance?

**Review by PikaJoyce — APPROVED** [review]
Yay! Awesome, just a small comment!

**Comment by makarandp0** [comment]
LGTM!

**Review by makarandp0 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (manjeshbhargav) [reply package.json:148]: @PikaJoyce It will be eventually when we release 5.0.0.

**Pull request merged**